examples/ui/anchor_layout.rs

```diff
@@ -0,0 +1,164 @@
+//! Shows an "anchor layout" style of ui layout
+use bevy::prelude::*;
+
+const MARGIN: Val = Val::Px(12.);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                title: "Bevy Anchor Layout Example".to_string(),
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_systems(Startup, spawn_layout)
+        .run();
+}
+
+fn spawn_layout(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+    commands.spawn(Camera2d);
+    commands
+        .spawn((
+            Node {
```

Comment:
It might be more natural to make this a 3x3 grid node, then you wouldn't need to spawn an intermediate node to layout each row.

Reply:
It could also just be a single node without rows or columns at all, with each of the children using `PositionType::Absolute`.